pkg/model/crd.go

```diff
@@ -718,6 +718,119 @@ func (r *CRD) SortedFieldNames() []string {
 	return fieldNames
 }
 
+// GetIgnoreTagging returns whether ensuring controller tags should be ignored
+// for a resource or not.
+func (r *CRD) GetIgnoreTagging() bool {
```

Comment:
After this refactor https://github.com/aws-controllers-k8s/code-generator/pull/325, I was thinking we move any config-accessing logic outside of model and into `config`. Would you be opposed to following that style or is there something I'm missing why it needs to be implemented here?